README.md

```diff
@@ -1,6 +1,37 @@
 Chitter Challenge
 =================
 
+Running:
+-------
+
+Clone this repository, then run:
+```bash
+bundle install
+```
+This will install the required gems
+
+To create the tables run:
+```bash
+createdb chitter
+psql -h 127.0.0.1 chitter < seeds/chitter_table.sql
+
+# (optional) seed the tables:
+psql -h 127.0.0.1 chitter < seeds/seeds.sql
+```
+
+To run the tests repeat the tables incuding seeding but name the database 'chitter_test. Then run:
+```bash
+rspec
+```
+
+To start the server run:
+```bash
+rackup
+```
+
+The website can then be found at: https://localhost:9292
+-------
+
```

Comment:
Nice and clear introduction. Will be better if included some screenshots of how the app works,